network/p2p/peerManager_test.go

```diff
@@ -53,58 +53,43 @@ func (suite *PeerManagerTestSuite) TestUpdatePeers() {
 
 	// create the connector mock to check ids requested for connect and disconnect
 	connector := new(mocknetwork.Connector)
-	connector.On("ConnectPeers", suite.ctx, mock.AnythingOfType("flow.IdentityList")).
+	connector.On("UpdatePeers", suite.ctx, mock.AnythingOfType("flow.IdentityList")).
```

Comment:
`ConnectPeers` -> `UpdatePeers`
And
Removed `DisconnectPeers` 
everywhere in this file